pages/documentation/simulation/powerflow/hades2.md

```diff
@@ -82,7 +82,7 @@ this parameter is `1`.
 
 **dcUseTransformerRatio**  
 The `dcUseTransformerRatio` property is an optional property that defines if ratio of transformers should be used in the
-flow equations. The default value of this parameter is `false`.
+flow equations. The default value of this parameter is `true`.
```

Comment:
I checked on hades2-integration code, it is `false` by default. Did you make a PR that changed the code? If yes, maybe specify the old and new default value and the version of the change.

Reply:
Just seen your PR, I still think it would be good to specify the old/new values and the version number of the change: `The default value of this parameter is true. It used to be false until the version X.X.X of hades2-integration.`

Reply:
Done!